server/server.js

```diff
@@ -34,6 +34,42 @@ const pool = new Pool({
 })
 
 // Express routes
+app.get("/api/guinea_pigs", (req, res) =>  {
```

Comment:
Make sure you are using the pattern of `api/v1/{whatever}` for your API endpoints.